auth_jwt_test.go

```diff
@@ -312,17 +311,97 @@ func TestLoginHandler(t *testing.T) {
 		})
 }
 
-func performRequest(r http.Handler, method, path string, token string) *httptest.ResponseRecorder {
```

Comment:
I removed this function because it was unused, according to gopls. I'll put it back if its needed.